src/main/resources/changelog.xml

```diff
@@ -971,5 +971,41 @@
 			</column>
 		</addColumn>
 	</changeSet>
+	
+	<changeSet id="add_issues" author="Aron Zwaan">
+		<createTable tableName="repository_issues">
+			<column name="issue_id" type="bigint" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
```

Comment:
Issue numbers are shared with pull requests, and start counting at 1 for each repository. Therefore the primary key is a compound key between `issue_id` and `repository_id`. See also the migration for pull requests.